doc/Developing.md

```diff
@@ -12,7 +12,7 @@ The following instructions are for Linux and macOS.
 For Windows instructions [click here](#building-on-windows).
 
 ## Prerequisites
-* [Yarn package manager](https://yarnpkg.com/en/docs/install)
+* https://yarnpkg.com/en/docs/install
```

Comment:
Agree, this file should not be in this feature proposal, file removed from the push request.